### PR TITLE
Feat/ha/catalog admin deploy to prod

### PR DESCRIPTION
**Pull request by hegeaal**

### CODE REVIEW

File: deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  labels:
+    app: catalog-admin-frontend
+spec:
+  template:
+    spec:
+      containers:
+        - name: catalog-admin-frontend
+          env:
+            - name: NEXTAUTH_URL
+              value: https://catalog-admin.fellesdatakatalog.digdir.no
```
Comment (valosnah) [inline deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml:15]: catalog-admin.demo.fellesdatakatalog

File: deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            - name: NEXTAUTH_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: catalog-admin-frontend-demo
```
Comment (valosnah) [inline deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml:19]: name: catalog-admin-frontend

File: deploy/prod/catalog-admin-frontend/catalog-admin-frontend-env.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    spec:
+      containers:
+        - name: catalog-admin-frontend
+          env:
+            - name: NEXTAUTH_URL
+              value: https://catalog-admin.fellesdatakatalog.digdir.no
+            - name: NEXTAUTH_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: catalog-admin-frontend-prod
```
Comment (valosnah) [inline deploy/prod/catalog-admin-frontend/catalog-admin-frontend-env.yaml:19]: name: catalog-admin-frontend

**Review by valosnah — CHANGES_REQUESTED** [review]

File: deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml (hunk 1 above)
Comment (hegeaal) [reply deploy/demo/catalog-admin-frontend/catalog-admin-frontend-env.yaml]: Fikset ✅ Antok at det gjelder for concept-catalog også?

**Review by valosnah — APPROVED** [review]

**Pull request merged**